src/goto-symex/shadow_memory_util.cpp

```diff
@@ -71,3 +78,108 @@ void clean_pointer_expr(exprt &expr, const typet &type)
   }
   POSTCONDITION(expr.type().id() == ID_pointer);
 }
+
+void log_get_field(
+  const namespacet &ns,
+  const messaget &log,
+  const irep_idt &field_name,
+  const exprt &expr)
+{
+  log.conditional_output(
+    log.debug(), [ns, field_name, expr](messaget::mstreamt &mstream) {
+      mstream << "Shadow memory: get_field: " << id2string(field_name)
```

Comment:
⛏️  `get_field` -> `get field` to match the other messages

Reply:
I actually changed the others to reflect the variable name

Reply:
No, it's ok that it correctly refers to the variable.
I changed the other accordingly instead.